dockstore-integration-testing/src/test/java/io/dockstore/client/cli/GeneralET.java

```diff
@@ -465,4 +525,124 @@ public void testLocalLaunchCWL() {
+                final Long tagID = testingPostgres.runSelectStatement("select tagid from tool_tag where toolid = "+toolID, new ScalarHandler<>());
+                final String path = testingPostgres.runSelectStatement("select cwlpath from tag where id = "+tagID, new ScalarHandler<>());
+                Assert.assertTrue("the cwl path should be changed to /test1.cwl", path.equals("/test1.cwl"));
+        }
+
+        /**
+        * Test to update the default path of WDL and it should change the tag's WDL path in the database
+        * @throws IOException
+        * @throws TimeoutException
+        * @throws ApiException
+        */
+        @Test
+        public void testUpdateToolPathWDL() throws IOException, TimeoutException, ApiException {
+                // Set up webservice
+                ApiClient client = getWebClient();
+
+                //Set up user api and get the container api
+                UsersApi usersApi = new UsersApi(client);
+                final Long userId = usersApi.getUser().getId();
+                usersApi.refresh(userId);
+                ContainersApi toolsApi = new ContainersApi(client);
+
+                // Make publish request (true)
+                final PublishRequest publishRequest = new PublishRequest();
+                publishRequest.setPublish(true);
+
+                //register tool
+                DockstoreTool c = getContainer();
+                DockstoreTool toolTest = toolsApi.registerManual(c);
+                toolsApi.refresh(toolTest.getId());
+
+                toolTest.setDefaultWdlPath("/test1.wdl");
+                toolsApi.updateTagContainerPath(toolTest.getId(),toolTest);
+                toolsApi.refresh(toolTest.getId());
+
+                // Set up DB
+                final CommonTestUtilities.TestingPostgres testingPostgres = getTestingPostgres();
+
+                //check if the workflow versions have the same workflow path or not in the database
+                final Long toolID = testingPostgres.runSelectStatement("select id from tool where name = 'testUpdatePath'", new ScalarHandler<>());
+                final Long tagID = testingPostgres.runSelectStatement("select tagid from tool_tag where toolid = "+toolID, new ScalarHandler<>());
+                final String path = testingPostgres.runSelectStatement("select wdlpath from tag where id = "+tagID, new ScalarHandler<>());
+                Assert.assertTrue("the cwl path should be changed to /test1.wdl", path.equals("/test1.wdl"));
+        }
+
+        /**
+         * Test to update the default path of Dockerfile and it should change the tag's dockerfile path in the database
+         * @throws IOException
+         * @throws TimeoutException
+         * @throws ApiException
+         */
+        @Test
+        public void testUpdateToolPathDockerfile() throws IOException, TimeoutException, ApiException {
+                // Set up webservice
+                ApiClient client = getWebClient();
```

Comment:
This section of the tests seem highly repetitive, try to refactor so it is easier to pick out the differences between the three test cases